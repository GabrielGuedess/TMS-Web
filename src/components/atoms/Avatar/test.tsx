import { render } from '@testing-library/react';

import { Avatar } from '.';

describe('<Avatar />', () => {
  it('should render the avatar correctly', () => {
    const { container } = render(
      <Avatar
        alt="Gabriel"
        src="https://avatars.githubusercontent.com/u/64827875?v=4"
      />,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
