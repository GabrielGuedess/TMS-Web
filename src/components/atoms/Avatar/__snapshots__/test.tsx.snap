// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render the avatar correctly 1`] = `
<div
  class="relative w-10 h-10 select-none overflow-hidden rounded-full bg-porcelain-50 dark:bg-smoke-950 transition-all"
>
  <img
    alt="Gabriel"
    class="select-none object-cover"
    data-nimg="fill"
    decoding="async"
    fetchpriority="high"
    sizes="(max-width: 768px) 100vw, 33vw"
    src="/_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=3840&q=75"
    srcset="/_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F64827875%3Fv%3D4&w=3840&q=75 3840w"
    style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
  />
</div>
`;
